dummy_tests.py

```diff
@@ -0,0 +1,15 @@
+import pytest
```

Comment:
lets put this in the tests.py file of ond of the apps so we know those work - and perhaps put in a real test instead of a dummy one?

Reply:
We can move the test to one of the apps' tests.py file, but I think it's better to stick with a dummy test for now because we don't really have any model functions currently, so there isn't anything we can test (correct me if I'm wrong), and I don't think adding a new model function is really related to this PR.

Reply:
I could argue that having a pure "installation" PR is a bit wasteful - better implement a small peice of functionality and the test for it and add what you need to run the test along the way.

But yeah moving the test can be enough, you know you'll remove it later anyway, and this way you can show me that you made sure the app test files work.

Reply:
Alright, I moved the test already, can we merge it then?